gpu_cluster/controllers/gpu_container_controller.py

```diff
@@ -7,6 +7,7 @@ class GPUContainerController(ContainerController):
 
     def __init__(self, config):
         super().__init__(config)
+        self.config = config;
```

Comment:
semicolon

Reply:
Can you also move this into the controller base class since both the CPU and GPU versions will need this?

Reply:
Fixed